src/r_vector/vectors.rs

```diff
@@ -279,6 +279,21 @@ impl From<Vec<String>> for Vector {
     }
 }
 
+impl Into<Vec<String>> for Vector {
+    fn into(self) -> Vec<String> {
+        match self {
+            Vector::Character(v) => v
+                .iter()
+                .map(|x| match x {
+                    OptionNA::Some(val) => val.clone(),
+                    OptionNA::NA => "NA".to_string(),
+                })
+                .collect(),
+            _ => todo!(),
```

Comment:
If we already tried to coerce to `character`, then we can replace this with 

```suggestion
            _ => unreachable!(),
```